inference/cluster_store.py

```diff
@@ -1,18 +1,91 @@
 # Cluster API
+import math
+import statistics
+
+from sklearn.cluster import KMeans
+import numpy as np
+from sklearn.preprocessing import MinMaxScaler
+
 
 class ClusterStore(object):
 
     def __init__(self, data, entity_map):
-        self.clusters = data
+        self.original_data = data
         self.entity_map = entity_map
+        self.clusters = None
+        self.total_cluster = 0
         self.clustered = False
         self.avg_similarity = 0.0
         self.avg_distance = 0.0
     
     # Function to run kmeans clustering on data 
     def build_kmeans_clusters(self):
-        # TODO
-        pass
+        '''
+        Build optimal k-means clusters based on minimum WCSS (Maximize distance) on clusters
+        :store: Total number of cluster, Clusters data
+        :update: clustered flag
+        '''
+        X = self.original_data
+        mms = MinMaxScaler()
+        mms.fit(X)
+        data_transformed = mms.transform(X)
+        # print(data_transformed)
```

Comment:
Please remove all commented out print statements. Keeps comments to documentation alone.

Reply:
Is it problem if the print located outside the class (in main function)?